config/default.yml

```diff
@@ -867,6 +867,16 @@ RSpec/Capybara/FeatureMethods:
   VersionChanged: '2.0'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Capybara/FeatureMethods
 
+RSpec/Capybara/NegationMatcher:
+  Description: Enforces use of `have_no_*` or `not_to` for negated expectations.
+  Enabled: pending
+  VersionAdded: '2.14'
+  EnforcedStyle: have_no
```

Comment:
With enforced style `have_no`:
```
...
    expect(page).not_to have_selector("div[data-hook='taxon_children']")
                 ^^^^^^^^^^^^^^^^^^^^
57497 files inspected, 1569 offenses detected, 1569 offenses autocorrectable
```
with `not_to`:
```
...
      expect(find('.payment-info')).to have_no_css('img')
                                    ^^^^^^^^^^^^^^
57497 files inspected, 205 offenses detected, 205 offenses autocorrectable
```

Despite my subjective personal preference, I'd suggest setting the default enforced style to what is causing fewer offences, unless we're enforcing some recommendation from Capybara to prefer `have_no_*` matchers.

I could find this:

> Capybara's RSpec matchers, however, are smart enough to handle either form. The two following statements are functionally equivalent:
```
expect(page).not_to have_xpath('a')
expect(page).to have_no_xpath('a')
```

Reply:
Indeed it is. We made sure to default to `not_to`.